src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -275,13 +294,12 @@ AND NOT EXISTS (select prs from p.results prs where prs.assessor.id = p.student.
     @Query("""
             SELECT distinct p FROM StudentParticipation p LEFT JOIN FETCH p.submissions s LEFT JOIN FETCH s.results r
                 WHERE p.exercise.id = :#{#exerciseId}
+                AND (p.testRun = FALSE OR p.testRun IS NULL)
```

Comment:
```suggestion
                AND p.testRun = FALSE
```

Reply:
fixed in 139c8fb547deadc2b10c0e497d91389dcb5f736c